neomodel/exception.py

```diff
@@ -1,6 +1,28 @@
 import importlib
 
 
+class ModelDefinitionMismatch(Exception):
+    """
+    Raised when it is impossible to resolve a Neo4j driver Node to a 
+    specific object. This can happen as a result of a query returning 
+    nodes for which class definitions do exist but have not been loaded 
+    (or rather imported) or because the retrieved nodes contain more 
+    labels for a known class.
+    
+    In either of these cases the mismatch must be reported
+    """
+    
+    def __init__(self, db_node, current_node_class_registry):
+        """
+        Initialises the exception with the database node that failed to resolve 
```

Comment:
this sentence doesn't express anything that isn't obvious and i'd consider it boilerplate.

Reply:
Indeed.